utilities/metadata.py

```diff
@@ -71,6 +70,58 @@
     # annotation_midbrainIncluded_rootdir = '/home/yuncong/CSHL_data_labelings_losslessAlignCropped_midbrainIncluded/'
     annotation_midbrainIncluded_v2_rootdir = '/home/yuncong/CSHL_labelings_v3/'
     patch_features_rootdir = '/home/yuncong/CSHL_patch_features_Sat16ClassFinetuned_v2'
+elif hostname.startswith('ip'):
+    print 'Setting environment for AWS compute node'
+    on_aws = True
```

Comment:
Change to `ON_AWS` - Let's capitalize all env./meta variables from now on. Previously I did not pay much attention to this, but I think it is good to start doing that for new variables. This makes it easier to know which are the meta variables in code.